LOCAL_VIRTUALENV.rst

```diff
@@ -163,6 +163,31 @@ Activate your virtualenv, e.g. by using ``workon``, and once you are in it, run:
     cd airflow/www
     yarn build
 
+Developing Providers
+--------------------
+
+In Airflow 2.0 we introduced split of Apache Airflow into separate packages - there is one main
+apache-airflow package with core of Airflow and 70+ packages for all providers (external services
+and software Airflow can communicate with).
+
+Developing providers is part of Airflow development, but when you install airflow as editable in your local
+development environment, the corresponding provider packages will be also installed from PyPI. However, the
+providers will also be present in your "airflow/providers" folder. This might lead to confusion,
+which sources of providers are imported during development. It will depend on your
+environment's PYTHONPATH setting in general.
+
+In order to avoid the confusion, you can set ``INSTALL_PROVIDERS_FROM_SOURCES`` environment to ``true``
+before running ``pip install`` command:
+
+.. code-block:: bash
+
+  INSTALL_PROVIDERS_FROM_SOURCES="true" pip install -U -e ".[devel,<OTHER EXTRAS>]" \
```

Comment:
With an editable install (`-e`) does the INSTALL_PROVIDERS_FROM_SOURCES flag do anything at all, as pip/setuptools doesn't copy the files, but install a "link" to the airflow tree (or so I thought)

Reply:
Yes. If you do not provide the variable it will also install providers from PyPI and you end up with two versions of providers - one in sources and one installed via packages.

Reply:
It is even described in the paragraph above actually.